cmd/ipfs/dist/install.sh

```diff
@@ -15,6 +15,7 @@ is_write_perm_missing=""
 for raw in $binpaths; do
   # Expand the $HOME variable.
   binpath=$(eval echo "$raw")
+  mkdir -p $binpath
```

Comment:
```suggestion
  mkdir -p "$binpath"
```

(need to quote in case there's a space in the path)

Reply:
```suggestion
  mkdir -p ${binpath}
```
Use `${}` syntax in case there is a space and a quote in the path.

Reply:
The expansion needs to be quoted to protect against spaces, even when using curly braces. In this case, the curly braces are equivalent and just a style preference. So it needs to be either `"$binpath"` or `"${binpath}"` (I don't really care which), but not `$binpath` nor `${binpath}`.